dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -865,7 +865,7 @@ private String createAdaptiveExtensionClassCode() {
 
                 boolean hasInvocation = false;
                 for (int i = 0; i < pts.length; ++i) {
-                    if (pts[i].getName().equals("org.apache.dubbo.rpc.Invocation")) {

```

Comment:
my understanding is if pts[i] is not null. getName will not be null.

Could you explain more what's scenario will cause NullPointException?